.github/workflows/build.yml

```diff
@@ -0,0 +1,147 @@
+  tests:
+    runs-on: ubuntu-18.04
+
+    name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ "7.4", "8.0" ]
+        symfony: [ "^4.4", "^5.2" ]
+        sylius: [ "~1.10.0", "~1.11.0" ]
+        node: [ "10.x" ]
+        mysql: [ "8.0" ]
+
+        exclude:
+          - sylius: ~1.10.0
+            symfony: 4.4
+
+          - sylius: ~1.11.0
+            php: 7.4
+
+    env:
+      APP_ENV: test
+      DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?serverVersion=${{ matrix.mysql }}"
+
+    steps:
+      -
+        uses: actions/checkout@v2
+
+      -
+        name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "${{ matrix.php }}"
+          ini-values: date.timezone=Europe/Warsaw, opcache.enable=1, opcache.enable_cli=1, opcache.memory_consumption=256, opcache.max_accelerated_files=32531, opcache.interned_strings_buffer=8, opcache.validate_timestamps=0, opcache.save_comments=1, opcache.fast_shutdown=0
+          extensions: intl, gd, opcache, mysql, pdo_mysql, :xdebug
+          tools: symfony
+          coverage: none
+
+      -
+        name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ matrix.node }}"
+
+      -
+        name: Shutdown default MySQL
+        run: sudo service mysql stop
+
+      -
+        name: Setup MySQL
+        uses: mirromutth/mysql-action@v1.1
+        with:
+          mysql version: "${{ matrix.mysql }}"
+          mysql root password: "root"
+
+      -
+        name: Output PHP version for Symfony CLI
+        run: php -v | head -n 1 | awk '{ print $2 }' > .php-version
+
+      -
+        name: Install certificates
+        run: symfony server:ca:install
+
+      -
+        name: Run Chrome Headless
+        run: google-chrome-stable --enable-automation --disable-background-networking --no-default-browser-check --no-first-run --disable-popup-blocking --disable-default-apps --allow-insecure-localhost --disable-translate --disable-extensions --disable-web-security --no-sandbox --enable-features=Metal --headless --remote-debugging-port=9222 --window-size=2880,1800 --proxy-server='direct://' --proxy-bypass-list='*' http://127.0.0.1 > /dev/null 2>&1 &
+
+      -
+        name: Run webserver
+        run: (cd tests/Application && symfony server:start --port=8080 --dir=public --daemon)
+
+      -
+        name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      -
+        name: Cache Composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-php-${{ matrix.php }}-composer-${{ hashFiles('**/composer.json **/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-php-${{ matrix.php }}-composer-
```

Comment:
From which repository did you take this file? There's no Symfony and Sylius restrict. Lines 105-114 in this file https://github.com/BitBagCommerce/SyliusCmsPlugin/blob/master/.github/workflows/build.yml#L105

Reply:
It's a file from our [DHL24PlShippingExportPlugin](https://github.com/BitBagCommerce/SyliusDHL24PlShippingExportPlugin/blob/master/.github/workflows/build.yml)
Changed - added sylius and symfony restrict.